### ISSUE TITLE
Unable to create annotations in latest CVAT release (probable incorrect DB value fom UI sent to cvat server)

### ISSUE THREAD

**Issue by anugrahasinha** [openvinotoolkit/cvat#4003]
### My actions before raising this issue
- [X] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [X] Searched [past issues](/issues)

### Expected Behaviour

Should be able to save the annotations.

### Current Behaviour

Unable to save annotations. 
Below error seen in browser console
```
Error: Request failed with status code 400. "null value in column \"rotation\" violates not-null constraint\nDETAIL:  Failing row contains (12, 0, 0, rectangle, f, 0, 224.0634765625,223.552734375,355.55284118652344,326.318695068359..., 2, 1, manual, null).\n".
```

### Possible Solution

NA

### Steps to Reproduce (for bugs)

1. Attaching the image details for CVAT_UI docker image and CVAT_server docker image.
2. cvat setup done after setting up CVAT_HOST environment variable.
3. Used standard method to install docker images for cvat.
4. Tried annotation of an image (from admin account)
5. error mentioned above seen, when trying to save annotation.

[cvat_server_image_inspect_details.txt](https://github.com/openvinotoolkit/cvat/files/7678486/cvat_server_image_inspect_details.txt)
[cvat_ui_image_inspect_details.txt](https://github.com/openvinotoolkit/cvat/files/7678487/cvat_ui_image_inspect_details.txt)

### Your Environment

- Git hash commit (`git log -1`):

```
commit f313a710015fa29327f69bb918dc4f6bd0f7bf6e (HEAD -> develop, origin/develop, origin/HEAD)
Author: Dmitry Kruchinin <dmitryx.kruchinin@intel.com>
Date:   Tue Dec 7 15:49:06 2021 +0300
    Update case 60 for check PR 3931 (#3993)
    Co-authored-by: dvkruchinin <dvkruchinin@gmail.com>
```

- Docker version `docker version` (e.g. Docker 17.0.05): `20.10.11`
- Are you using Docker Swarm or Kubernetes? `normal docker container on Ubuntu`
- Operating System and version (e.g. Linux, Windows, MacOS): `Linux : Ubuntu 18.04`
- Code example or link to GitHub repo or gist to reproduce problem: NA
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

**Comment by bsekachev** [comment]
@anugrahasinha 
Please, send us request details. Probably you use outdated version of client (it might be cached by your browser for example).

**Comment by anugrahasinha** [comment]
@bsekachev 

I hope below adds more information to the issue I am facing.
`Request URL : http://172.30.63.156:8080/api/v1/jobs/2/annotations?action=create`

`Request Payload : {"shapes":[],"tracks":[{"frame":0,"label_id":1,"group":0,"source":"manual","attributes":[],"shapes":[{"type":"rectangle","occluded":false,"z_order":0,"points":[138.6279296875,213.7890625,462.33709716796875,380.70159912109375],"outside":false,"attributes":[],"frame":0}]}],"tags":[],"version":0}`

`Response : "null value in column \"rotation\" violates not-null constraint\nDETAIL:  Failing row contains (rectangle, f, 0, 138.6279296875,213.7890625,462.33709716796875,380.70159912109375, 2, 0, f, 2, null).\n"`

Does the above help?

**Comment by bsekachev** [comment]
There is not rotation property in your request. 
What version of client can you see in UI? (User menu > About)

**Comment by anugrahasinha** [comment]
> There is not rotation property in your request. What version of client can you see in UI? (User menu > About)

Server version: 1.7
Core version: 3.17.0
Canvas version: 2.8.0
UI version: 1.25.0

**Comment by bsekachev** [comment]
Core version should be at least 3.19, UI 1.26.0.
Be sure you have updated CVAT correctly, try to perform installation steps one more time and if it does not help send us all steps you did.

**Comment by anugrahasinha** [comment]
Problem persists
Steps executed below
[cvat_not_working_reinstallation_steps.txt](https://github.com/openvinotoolkit/cvat/files/7692701/cvat_not_working_reinstallation_steps.txt)

**Comment by bsekachev** [comment]
Try:

``docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d --build``

Also you do not need to remove any containers/images before installation. Do this command instead:

``docker-compose down``

**Comment by anugrahasinha** [comment]
@bsekachev 
The problem got resolved after recompiling the whole thing. Thanks.

However, the version numbers of still the same.
I think this isnt write, as after following the manual, once would clone from git, and perform `docker-compose up -d`
and this has issues. A recompilation is necessary.

I think the docker image should be updated with the latest compilation. Would like to add someone else from the team who can update the latest image on docker-hub?

**Comment by bsekachev** [comment]
As far as I know the latest image on docker-hub get updated automatically on releases. We do not push develop versions there. 
I am not sure how you could get develop version of server together with release version of client (actually ``docker-compose up`` (without ``-f docker-compose.dev.yml``) does fetch the latest client and server from docker hub. And the problem does not look common, because you are the first who contacted us with it.

**Comment by anugrahasinha** [comment]
@bsekachev 
I understand, thanks for detailing this out.
As I said, I would request you to please ask the concerned team/member to check this once again, as a fresh install.

**Comment by azhavoro** [comment]
@anugrahasinha as you can see https://hub.docker.com/r/openvino/cvat_server/tags the latest published docker image is 1.7.0 which published a month ago, so it's absolutely impossible to download `cvat_server` image for the develop from the docker hub. The only way to do this is by building locally.
I will close the issue
